spiritual-care.csl

```diff
@@ -0,0 +1,653 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="de-DE" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>SPIRITUAL CARE</title>
+    <title-short>SC</title-short>
+    <id>http://www.zotero.org/styles/spiritual-care</id>
+    <link href="http://www.zotero.org/styles/spiritual-care" rel="self"/>
+    <link href="http://www.zotero.org/styles/apa" rel="template"/>
+    <link href="https://" rel="documentation"/>
```

Comment:
Change to ` <link href="https://www.degruyter.com/journal/key/spircare/html" rel="documentation"/>` and add a second link with the following URL: https://www.degruyter.com/publication/journal_key/SPIRCARE/downloadAsset/SPIRCARE_Instructions%20for%20Authors--en.pdf

Reply:
Done